src/gettingstarted/local/docksal.md

```diff
@@ -0,0 +1,80 @@
+# Using Docksal for local development
+
+[Docksal](https://docksal.io) is a docker based local development tool that plays nicely with Platform.sh.  It is maintained by a community of developers but is a viable option for most Platform.sh projects.
+
+See the [Docksal documentation](https://docs.docksal.io/) for installing and setting up Docksal on your system.
+
+Docksal will ask you to create a `.docksal` directory in your application root, which functions similarly to the `.platform.app.yaml` file. It is safe to check this directory into your Git repository as Platform.sh will simply ignore it.
+
+All interaction with a Docksal project will happen through the `fin` command. For more info you can type the `help` subcommand to get all available commands and options.
+
+```bash
+fin help
+```
+
+## Using Platform.sh CLI within Docksal
+
+To start using the Platform.sh CLI within your docksal project. The `SECRET_PLATFORMSH_CLI_TOKEN` will need to be set. This is an API Token found with your Platform.sh Account and can be generated by going to the [API Tokens](https://accounts.platform.sh/user/api-tokens) page and clicking the `Create API Token` link. This will allow for you to interact with your Platform.sh account from within the CLI container.
+
+```bash
+fin config set --global SECRET_PLATFORMSH_CLI_TOKEN=XXX
+```
+
+## Pulling a Platform.sh project
+
+The Docksal CLI or PHP image ships with the Platform.sh CLI tool. To use the tool and pull a project locally, make sure you have uploaded your SSH key to your Platform.sh account. Once that has been done and a `SECRET_PLATFORMSH_CLI_TOKEN` has been added using the above step. The following command can be ran. Replace `PROJECT_ID` with your project's ID, which can found within the Platform.sh dashboard. Replace `PROJECT_DIRECTORY` with the name of the local directory you'd like the project cloned into. With your SSH key and the `SECRET_PLATFORMSH_CLI_TOKEN` set, you can execute the following command.
+
+If you do not already have Platform.sh CLI installed locally you can use the one that has been installed of the CLI image. The advantage of this would mean that the tool would never have to be installed locally and therefore is one less of a dependency.
```

Comment:
It may be worth mentioning here that simply downloading the project doesn't add Docksal configuration to it.  You still have to do that yourself.  That threw me for a loop the first time I tried it.